src/cls/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ if (WITH_RBD)
 
   set(cls_rbd_client_srcs rbd/cls_rbd_client.cc rbd/cls_rbd_types.cc)
   add_library(cls_rbd_client STATIC ${cls_rbd_client_srcs})
-  target_link_libraries(cls_rbd_client cls_lock_client)
+  target_link_libraries(cls_rbd_client PRIVATE librados cls_lock_client)
```

Comment:
Why is `librados` getting pulled into tons of random libraries? This seems less than ideal.